### DISCUSSION TITLE
账号绑定成功后，列表不显示贴吧

### DISCUSSION THREAD

**Discussion by spring1hibernate** [MoeNetwork/Tieba-Cloud-Sign#191] [疑问]
你已经成功绑定了一个贴吧账号，但是还没有添加任何贴吧，点击上方的按钮扫描贴吧。
扫描不到贴吧？请检查百度账号BDUSS是否已经失效（尝试重新绑定）

刷新也不行，手动绑定和扫码绑定都是一样的回显。

求大神解决，已经是最新版！

![3](https://user-images.githubusercontent.com/112772927/188280518-e64c4620-6487-46b3-b1c6-637b5d7a27e8.png)
![1](https://user-images.githubusercontent.com/112772927/188280469-b472bd20-2bc3-427d-861c-0fe732ec62bb.png)
![2](https://user-images.githubusercontent.com/112772927/188280475-e7947e80-0bd4-4120-ac8c-fc4a582dc59b.png)

**Comment by n0099** [comment]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/discussions/186#discussioncomment-3522861
https://github.com/MoeNetwork/Tieba-Cloud-Sign/issues/188

**Reply by spring1hibernate** [reply]
按照方法修改了，还是不行！
